pkg/scheduler/objects/allocation_ask.go

```diff
@@ -51,10 +52,17 @@ type AllocationAsk struct {
 	requiredNode     string
 	allowPreemption  bool
 	originator       bool
+	allocLog         map[string]*AllocationLogEntry
 
 	sync.RWMutex
 }
 
+type AllocationLogEntry struct {
+	Message   string
+	Timestamp time.Time
```

Comment:
Timestamp is ambiguous. If count > 1 the timestamp is linked to the latest occurrence. That either needs to show in the name or we need two: first  occurrence and most recent occurrence timestamp.

Reply:
Renamed to `LastOccurrence`.